packages/client/modules/meeting/components/MeetingAvatarGroup/NewMeetingAvatarGroup.tsx

```diff
@@ -185,15 +186,21 @@ export default createFragmentContainer(NewMeetingAvatarGroup, {
       team {
         id
         teamMembers {
```

Comment:
Passing `teamMembers` down to `AddTeamMemberAvatarButton` and then to `AddTeamMemberModal` as we want to check whether the user is inviting a new team member rather than a new meeting member